lib/ts/index.ts

```diff
@@ -15,26 +15,14 @@
 
 import SuperTokens from "./supertokens";
 import SuperTokensError from "./error";
-import * as express from "express";
+import wrappers from "./wrappers";
 
 // For Express
 export default class SuperTokensWrapper {
     static init = SuperTokens.init;
 
     static Error = SuperTokensError;
 
-    static middleware() {
```

Comment:
Okay. But we need to deprecate this usage as this will be useful only for express framework. If any other framework's middleware is to be used, it will be required to import that middleware from that framework

Reply:
Yea ok.. You can add a @deprecated annotation to the middleware inside `SuperTokensWrapper` and outside. What does the import statement look like for a specific framework?